apps/postgresql-server/bin/initialize_db.sh

```diff
@@ -35,8 +35,11 @@ CREATE DATABASE mediacloud WITH
 EOF
 psql -v ON_ERROR_STOP=1 -c "${CREATE_DB_SQL}"
 
-# Initialize with schema
-psql -v ON_ERROR_STOP=1 -d mediacloud -f /opt/postgresql-server/schema/mediawords.sql
+# Run migrations with pgmigrate package
+cd /opt/mediacloud && pgmigrate -t latest migrate
+
+# Dump schema file for reference in development
+psql -v ON_ERROR_STOP=1 mediacloud -c '\! pg_dump mediacloud > /tmp/mediawords.sql'
```

Comment:
1. `/tmp` and `/var/tmp` could be tmpfs filesystems mounted by Docker to the container; I think it's better to store the generated schema somewhere less temporary, e.g. `/`;
2. Instead of running `pg_dump` from within `psql` (`psql`'s `\!` command just starts a shell), you can just run `pg_dump` directly;
3. By the way, by default `psql` just ignores errors that it encounters in the input SQL. For example, if you had the following SQL file:

    ```sql
    CREATE TABLE foo (name TEXT);
    
    blergh;
    
    CREATE TABLE bar (name TEXT);
    ```

    and were to run `psql -d database_name -f that_file_with_a_typo_in_the_middle.sql`, it would `CREATE TABLE foo`, complain about the `blergh` statement and then happily `CREATE TABLE bar`. This is something to be wary of when, for example, importing large dumps because one might end up with an incomplete imported dump.